src/MooLite/plugins/EnhancerHelper/EnhancerHelperPlugin.ts

```diff
@@ -0,0 +1,213 @@
+import { MooLitePlugin } from "src/MooLite/core/plugins/MooLitePlugin";
+import { MooLiteTab } from "src/MooLite/core/plugins/MooLiteTab";
+import { markRaw } from "vue";
+import EnhancerHelperPluginDisplay from "src/MooLite/plugins/EnhancerHelper/EnhancerHelperPluginDisplay.vue";
+import { ItemDetail } from "src/MooLite/core/inventory/items/ItemDetail";
+import { ItemAmount } from "src/MooLite/core/inventory/items/ItemAmount";
+import { ItemHrid } from "src/MooLite/core/inventory/ItemHrid";
+import { CharacterItem } from "src/MooLite/core/inventory/CharacterItem";
+
+export class EnhancerHelperPlugin extends MooLitePlugin {
+    name: string = "Enhancer Helper";
+    key = "enhancer-helper";
+    description: string = "Various enhancing tools - By Void";
+
+    tab: MooLiteTab = {
+        icon: "💎",
+        pluginName: this.name,
+        componentName: "EnhancerHelperPluginDisplay",
+        component: markRaw(EnhancerHelperPluginDisplay),
+    };
+
+    getEnhanceableItems(): ItemDetail[] {
+        return this._game.inventory.sortedAlphabeticalItems.filter((item) => item.enhancementCosts != null);
+    }
+
+    getEnhancementMaterials(itemHrid: ItemHrid): ItemAmount[] | undefined | null {
```

Comment:
I think this can be only `null`

Reply:
When I try to build it gives an error saying it can be undefined too /shrugs